app-shells/mksh/mksh-54.recipe

```diff
@@ -36,11 +36,9 @@ REQUIRES="
 BUILD_REQUIRES="
 	haiku_devel
 	"
-
 BUILD_PREREQUIRES="
 	cmd:awk
 	cmd:cmp
-	cmd:ed
```

Comment:
Error is:
unable to resolve prerequired packages for build for mksh-54
dependency-infos:
/boot/home/haikuports/repository/mksh-54.DependencyInfo
repositories:
[u'/boot/home/haikuports/packages', '/boot/system/packages']